.github/workflows/playwright.yml

```diff
@@ -0,0 +1,32 @@
+name: create-fuels e2e tests
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build
+        uses: ./.github/actions/test-setup
+        with:
+          node-version: ${{ matrix.env.version || 20 }}
+      - name: Run Fuel Node
+        run: (cd templates/nextjs && pnpm run fuels:dev) &
+      - name: Copy .env.example and create .env.local off it
+        run: cp templates/nextjs/.env.example templates/nextjs/.env.local
+      - name: Run Next.js server
+        run: (cd templates/nextjs && pnpm run dev) &
+      - name: Install Playwright Browsers
+        run: pnpm exec playwright install --with-deps
```

Comment:
Whats this doing? Can it not just be a dependency?

Reply:
@danielbate unfortunately not. When you install playwright, it does not install the browsers along with it